src/kafka/configuration-manager.ts

```diff
@@ -22,15 +24,17 @@ export class ConfigurationManager {
       kafkaHost: get('kafkaHost'),
       ssl: get('ssl'),
       sslOptions: get('sslOptions'),
-      sasl: get('sasl')
+      sasl: get('sasl'),
+      fetchMaxWaitMs: get('fetchMaxWaitMs') || 300,
+      fetchMinBytes: get('fetchMinBytes') || 0.5 * MB,
+      fetchMaxBytes: get('fetchMaxBytes') || 5 * MB
     };
   }
 
   get backpressureOptions(): BackpressureConfig {
     const get = this.getter(this.config.consumer);
     return {
       pause: (get<BackpressureConfig>('backpressure') || /* istanbul ignore next */{}).pause,
-      resume: (get<BackpressureConfig>('backpressure') || /* istanbul ignore next */{}).resume,
       topMB: (get<BackpressureConfig>('backpressure') || /* istanbul ignore next */{}).topMB
```

Comment:
why resume var is deleted?

Reply:
because i forgot to revert some develop changes